### PR TITLE
appVersion Updated

### PR DESCRIPTION
**Pull request by kundan2707**
#125

### CODE REVIEW

**Review by MrHohn — COMMENTED** [review]
@kundan2707 thanks for the update - could you squash the fix-up commit?
/approve

File: charts/cluster-proportional-autoscaler/Chart.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 apiVersion: v2
 name: cluster-proportional-autoscaler
 version: 1.0.0
```
Comment (MrHohn) [inline charts/cluster-proportional-autoscaler/Chart.yaml:3]: Actually - I assume this should be bumped as well?

Comment (krmichel) [reply charts/cluster-proportional-autoscaler/Chart.yaml:3]: I would make it 1.0.2

**Comment by kundan2707** [comment]
Closing as created new PR for this
https://github.com/kubernetes-sigs/cluster-proportional-autoscaler/pull/130

**Pull request closed without merging**